src/app/utils/handle-mongo-error.ts

```diff
@@ -0,0 +1,39 @@
+import { MongoError } from 'mongodb'
+import { Error as MongooseError } from 'mongoose'
+
+// Default error message if no more specific error
+const defaultErrorMessage = 'An unexpected error happened. Please try again.'
+
+export const getMongoErrorMessage = (
+  err?: MongoError | MongooseError | string,
```

Comment:
Should we instead move `defaultErrorMessage` here so callers of this method can override the default with their own (for increased flexibility)?

```ts
export const getMongoErrorMessage = (
  err?: MongoError | MongooseError | string,
  defaultErrorMessage = 'An unexpected error happened. Please try again.'
```

Reply:
If doing this, please add test cases for it too